UITests/README.md

```diff
@@ -1,29 +1,79 @@
 
-The test scenarios are written to test the controls in the Xaml Controls Gallery app, which contains all UWP Xaml controls. The Xaml Controls Gallery app needs to be installed on the machine where you are running your tests. Follow the instructions in the root README to install the app.
+The test scenarios are written to test the controls in the WinUI 3 Gallery app. The procedure below outlines the steps needed to build and deploy WinUI 3 Gallery and WinAppDriver to run the UI Tests. These steps mirror what azure-pipelines.yml does in pipeline runs.
 
+## Deploy WinUI 3 Gallery
 
-## Requirements
+The easiest way to deploy WinUI 3 Gallery for unit test execution is to simply build WinUIGallery\WinUIGallery.sln and F5 deploy it from within Visual Studio.  Alternatively, the following commands can be used for automation.
 
-- Windows 10 PC with the latest Windows 10 version (Version 1809 or later)
-- Microsoft Visual Studio 2017 or later
-- [Xaml Controls Gallery](../../XamlControlsGallery/XamlControlsGallery.sln) built and deployed 
+1. Generate the test signing certificate:
 
+```shell
+    PS> .\build\GenerateTestPfx.ps1
+```
 
-## Getting Started
+1. Build and publish the WinUI 3 Gallery from the command line, e.g.:
 
-1. Open `UITests.sln` in Visual Studio
-2. Select **Test** > **Windows** > **Test Explorer**
-3. Select **Run All** on the test pane or through menu **Test** > **Run** > **All Tests**
+```shell
+    dotnet.exe publish WinUIGallery\WinUIGallery.sln /p:AppxPackageDir=AppxPackages\ /p:platform=x64 /p:PublishProfile=./WinUIGallery/Properties/PublishProfiles/win10-x64.pubxml
+```
 
-> Once the project is successfully built, you can use the **TestExplorer** to pick and choose the test scenario(s) to run
+1. Locate the WinUI 3 Gallery package output folder from above and deploy for testing:
 
+```shell
+    PS> .\WinUIGallery\AppxPackages\WinUIGallery.Desktop_Test\Install.ps1
+```
 
-## Adding/Updating Test Scenario
+## Run WinAppDriver
+
+The test runner (vstest.console.exe and VS Test Explorer) should automatically launch WinAppDriver. Alternatively, it can be launched manually to observe diagnostic output as follows.
+
+1. Download and install the latest version of WinAppDriver from [here](https://github.com/microsoft/WinAppDriver/releases)
+
+1. Run WinAppDriver:
+
+```shell
+    C:\Program Files (x86)\Windows Application Driver>WinAppDriver.exe
+```
+
+## Build and run/debug Tests
+
+The easiest way to run/debug the UI tests is with Visual Studio, as follows:
+
+   * Open `UITests\UITests.sln` in Visual Studio
+   * Select **Test** > **Windows** > **Test Explorer**
+   * Select **Run All** on the test pane or through menu **Test** > **Run** > **All Tests**
+
+   Once the project is successfully built, you can use the **Test Explorer** to choose  test scenarios to run/debug
+
+Alternatively, the following commands can be used for automation:
+
+1. Build UITests:
+
+```shell
+    >msbuild UITests\UITests.sln
```

Comment:
```suggestion
    msbuild UITests\UITests.sln
```

Reply:
Also is it worth pointing out that msbuild is only available in the Developer Prompt and not usually available when opening a terminal, even is VS is installed? Running `dotnet build UITests\UITests.sln` should work as alternative.

Reply:
excellent